unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValuesTest.ts

```diff
@@ -170,5 +174,28 @@ export function DynamicFacetValuesTest() {
         expect(facet.showLessValues).toHaveBeenCalledTimes(1);
       });
     });
+
+    describe('when the facet option enableMoreLess is false', () => {
+      beforeEach(() => {
+        facet = DynamicFacetTestUtils.createFakeFacet({ numberOfValues: 5, enableMoreLess: false });
+        initializeComponent();
+      });
+
+      it(`when moreValuesAvailable is true
+      should not render the "Show less" button`, () => {
+        facet.moreValuesAvailable = true;
+
+        createValuesFromResponse();
+        expect(moreButton()).toBeFalsy();
+      });
+
+      it(`when clicking on the "Show more" button
```

Comment:
Hmm, I am not sure about this test description. If `enableMoreLess` is `false`, how would one click the `show more button`?

Should we just check that there is no "less" button rendered?

Reply:
oups good catch I'll have to change the description seems I copied the wrong one 😅